tests/Generator/SodiumRandomGeneratorTest.php

```diff
@@ -39,11 +39,11 @@ public function testFactoryUsesSodiumRandomGenerator()
         $uuidFactory->setRandomGenerator(new SodiumRandomGenerator());
         Uuid::setFactory($uuidFactory);
 
-        $uuid = Uuid::uuid4();
-
+        /** @var UuidFactory $actualUuidFactory */
+        $actualUuidFactory = Uuid::getFactory();
         $this->assertInstanceOf(
             SodiumRandomGenerator::class,
-            $uuid->getFactory()->getRandomGenerator()
+            $actualUuidFactory->getRandomGenerator()
```

Comment:
I'm not sure about this change, but originally it was calling the static method non-statically.